diff -Nur instead-1.5.2/icon/Makefile instead-1.5.2/icon/Makefile
--- instead-1.5.2/icon/Makefile	2011-10-16 15:18:51.000000000 +0300
+++ /tmp/1/instead-1.5.2/icon/Makefile	2011-11-10 22:37:51.312000181 +0200
@@ -6,6 +6,9 @@
 install:
 	$(INSTALLD) $(ICONPATH)
 	$(INSTALL) sdl_instead.png $(ICONPATH)
+	$(INSTALL) sdl_instead_wnd.png $(ICONPATH)
 
 uninstall:
 	$(RM) $(ICONPATH)/sdl_instead.png
+	$(RM) $(ICONPATH)/sdl_instead_wnd.png
+
diff -Nur instead-1.5.2/icon/Makefile.windows instead-1.5.2/icon/Makefile.windows
--- instead-1.5.2/icon/Makefile.windows	2011-10-16 15:18:51.000000000 +0300
+++ /tmp/1/instead-1.5.2/icon/Makefile.windows	2011-11-10 22:38:01.616000182 +0200
@@ -3,3 +3,4 @@
 install:
 	if not exist ..\bin\icon mkdir ..\bin\icon
 	copy sdl_instead.png ..\bin\icon
+	copy sdl_instead_wnd.png ..\bin\icon
diff -Nur instead-1.5.2/src/sdl-instead/graphics.c instead-1.5.2/src/sdl-instead/graphics.c
--- instead-1.5.2/src/sdl-instead/graphics.c	2011-10-16 15:18:49.000000000 +0300
+++ /tmp/1/instead-1.5.2/src/sdl-instead/graphics.c	2011-11-10 22:39:09.676000182 +0200
@@ -1358,7 +1358,7 @@
 #define ICON_PATH "./icon"
 #endif
 
-	icon = IMG_Load( ICON_PATH"/sdl_instead.png" );
+	icon = IMG_Load( ICON_PATH"/sdl_instead_wnd.png" );
 	if ( icon ) {
 		SDL_WM_SetIcon( icon, NULL );
 	}
